examples/SLAM/index.html

```diff
@@ -0,0 +1,184 @@
+  <head>
+    <script>
+      const API_TOKEN_XR8 = "sU7eX52Oe2ZL8qUKBWD5naUlu1ZrnuRrtM1pQ7ukMz8rkOEG8mb63YlYTuiOrsQZTiXKRe";
+    </script>
+    
+    <script>
+      const WEBXR_REQUIRED_FEATURES = [{{ webxrRequiredFeatures }}];
+      const WEBXR_OPTIONAL_FEATURES = [{{ webxrOptionalFeatures }}];
+    </script>
+
+    <link rel="preload" href="{{ settings.project.name }}-bundle.js?t={{timestamp}}" as="script">
+    <title>{{ settings.project.name }}</title>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="theme-color" content="{{ settings.runtime.xrButtonColor }}">
+    <style>
+    html {
+        height: 100%;
+    }
+    body {
+        height: 100%;
+        margin: 0px;
+        background-image: linear-gradient(black, #2b2b2b);
+    }
+    .xr-button-container {
+      display: block;
+      position: fixed;
+      width: 100px;
+      bottom: 100px;
+      left: 0;
+      right: 0;
+      margin: 0 auto;
+      justify-content: center;
+      pointer-events: none;
+    }
+
+    #vr-button,
+    #ar-button {
+      display: none;
+      pointer-events: all;
+      cursor: pointer;
+    }
+
+    #canvas {
+      -webkit-touch-callout: none;
+      -webkit-user-select: none;
+      -khtml-user-select: none;
+      -moz-user-select: none;
+      -ms-user-select: none;
+      user-select: none;
+      touch-action: none;
+    }
+
+    /* Once supported state is known, show the button */
+    [data-supported] {
+      width: 100px !important;
+      transition: 0.5s;
+    }
+
+    /* If not supported, we dim out */
+    [data-supported='0'] {
+      opacity: 0.25;
+    }
+
+    .xr-button-label {
+      fill: {{ settings.runtime.xrButtonColor }};
+    }
+
+    .xr-button-frame {
+      stroke: {{ settings.runtime.xrButtonColor }};
+    }
+
+    #version {
+      text-align: right;
+      position: absolute;
+      right: 10px;
+      bottom: 10px;
+      color: white;
+      font-size: 18pt;
+      font-family: monospace;
+      /*display: none; */
+      z-index: 99999;
+    }
+
+    #version a {
+      transition: opacity 0.5s;
+      opacity: 0;
+    }
+
+    #version:hover a{
+      opacity: 1;
+    }
+
+    </style>
+    <script>
+        var Module = {
+          webxr_framebuffer_scale_factor: {{ settings.vr.framebufferScaleFactor }}
+        };
+    </script>
+    {{ if settings.runtime.pwa.enable }}
+    <link rel="manifest" href="manifest.json">
+    <link rel="apple-touch-icon" href="homescreen192.png">
+    <script>
+        if('serviceWorker' in navigator) {
+            navigator.serviceWorker.register('service-worker.js')
+                .then(reg => console.log('Service worker registered!', reg))
+                .catch(err => console.log('Unable to register service worker!', err));
+        }
+    </script>
+    {{ end }}
+    <script src="wonderland.min.js?t={{timestamp}}"></script>
+    <script type="module">
+        function loadScript(scriptURL) {
```

Comment:
Heads-up that we will be changing this to only include the bundle (no more wonderland.min.js nor this module code, which moves into index.js) Ideally no more custom template is necessary and everything moves into index.js.